aiida/engine/processes/control.py

```diff
@@ -262,7 +262,7 @@ def handle_result(result):
                 # unwrap is need here since LoopCommunicator will also wrap a future
                 future = unwrap_kiwi_future(future)
                 result = future.result()
-            except communications.CommunicationTimeout:
+            except communications.TimeoutError:
```

Comment:
Just check the history, it used to be `from aiida.manage.external.rmq import CommunicationTimeout`.

Reply:
And there it used to be defined as `CommunicationTimeout = kiwipy.communications.TimeoutError  # pylint: disable=invalid-name` So this PR is restoring that behavior